### DISCUSSION TITLE
Kong Manager Network Error

### DISCUSSION THREAD

**Discussion by aureuslim** [Kong/kong-manager#219] [Q&A]
I have 2 Kong setup with Kong Manager enabled, 1 Kong OSS in Docker, and 1 Kong Enterprise Free Version in Ubuntu.

Both my Kong Manager works when using http://localhost.com/, however both has the same issue of having Network Error under the Workspaces overview.

![image](https://github.com/Kong/kong-manager/assets/49929225/6e3dee10-b45f-46af-b28c-c25b7b1852ac)

The configurations that I have edited in kong.conf are as follows:
admin_listen = 0.0.0.0:8001 reuseport backlog=16384, 0.0.0.0:8444 http2 ssl reuseport backlog=16384
admin_gui_listen = 0.0.0.0:8002, 0.0.0.0:8445 ssl
admin_gui_url = http://localhost:8002/
admin_gui_path = /
admin_gui_api_url = http://localhost:8002/

This is the error displayed on Google Chrome Console:

GET http://localhost:8002//default/kong net::ERR_CONNECTION_REFUSED
GET http://localhost:8002//workspaces?size=15&sort_by=created_at&counter=true net::ERR_CONNECTION_REFUSED

Can anyone please let me know if I missed out on any other important configurations or did I edit any of these configurations wrongly?

I can provide more details as to what I have done in the setup, or whatever necessary.

**Comment by ddodoo** [comment]
@aureuslim 

I first debugged for any errors using the web browser developer tool F12 

I kept getting errors with `http://public-ip/workspaces?size=1000`, **net::ERR_CONNECTION_TIMED_OUT**

I updated configuration file kong.conf with `admin_listen = 0.0.0.0:8001` and open security group rule port 8001 on AWS

I confirmed by using curl command 'curl http://public-ip:8001/workspaces?size=1000'

This resolved  my issue, I found this tool helpful because there wasn't much useful online https://www.phind.com/


Hope this helps :)
